src/uct/ib/ud/accel/ud_mlx5.h

```diff
@@ -13,10 +13,15 @@
 
 
 typedef struct {
-    uct_ud_ep_t                         super;
     uct_ib_mlx5_base_av_t               av;
     uint8_t                             is_global;
     struct mlx5_grh_av                  grh_av;
+} uct_ud_mlx5_ep_peer_address_t;
+
+
+typedef struct {
+    uct_ud_ep_t                         super;
+    uct_ud_mlx5_ep_peer_address_t       peer_address;
```

Comment:
`peer_address` is next after `uct_ud_ep_t` for both verbs and mlx5 eps. Do you really need the `ep_peer_offset` thru the code?

Reply:
done